packages/zen-bull-netting/src/NettingLib.sol

```diff
@@ -172,9 +172,20 @@ library NettingLib {
         returns (uint256)
     {
         uint256 bullTotalSupply = IERC20(zenBull).totalSupply();
-        uint256 crabAmount =
-            withdrawsToProcess * IZenBullStrategy(zenBull).getCrabBalance() / bullTotalSupply;
         (, uint256 crabDebt) = IZenBullStrategy(zenBull).getCrabVaultDetails();
-        return (crabAmount * crabDebt / IERC20(crab).totalSupply());
+        uint256 share = div(withdrawsToProcess, bullTotalSupply);
+        uint256 crabAmount = mul(share, IZenBullStrategy(zenBull).getCrabBalance());
+
+        return div(mul(crabAmount, crabDebt), IERC20(crab).totalSupply());
+    }
+
+    function mul(uint256 x, uint256 y) internal pure returns (uint256) {
```

Comment:
why not just use wmul and wdiv?

Reply:
@alpinechicken seems like same implementation if you look at strategy math vs above, can we just not use safe math but have the order of operations just be the same and get the same result?  it seems using the strategy math implementation does waste some gas due to overflow prevention, right?

Reply:
yeah also the StrategyMath is written with a diff solidity version, if I just import it, it will not work